homeassistant/components/sma_manager/SMA.py

```diff
@@ -0,0 +1,180 @@
+"""SMA Manager API"""
+
+#  Copyright (c) 2023.
+#  All rights reserved to the creator of the following script/program/app, please do not
+#  use or distribute without prior authorization from the creator.
```

Comment:
I don't know if still applies as you signed the CLA
https://www.home-assistant.io/developers/cla/

Reply:
CLA kinda voids it, but my editor is just set to auto add them.